IoTCloudSDK/IoTCloudSDK/IoTCloudAPI.swift

```diff
@@ -380,6 +382,23 @@ public class IoTCloudAPI: NSObject, NSCoding {
 
         return newIotapi
     }
+    
+    func saveToUserDefault(){
+        NSUserDefaults.standardUserDefaults().setObject(NSKeyedArchiver.archivedDataWithRootObject(self), forKey: "IoTCloudAPI")
+        NSUserDefaults.standardUserDefaults().synchronize()
+    }
+
+    public override func isEqual(object: AnyObject?) -> Bool {
```

Comment:
I see. I think it is not expected to be as public method. What about making it internal same as other  isEqual methods?